src/libraries/System.Reflection.Metadata/tests/Metadata/TagToTokenTests.cs

```diff
@@ -95,6 +95,7 @@ private IEnumerable<XxxTag> GetTags()
 
         [Fact]
         [ActiveIssue("https://github.com/dotnet/runtime/issues/50714", typeof(PlatformDetection), nameof(PlatformDetection.IsBuiltWithAggressiveTrimming), nameof(PlatformDetection.IsBrowser))]
+        [ActiveIssue("https://github.com/dotnet/runtime/issues/50714", typeof(PlatformDetection), nameof(PlatformDetection.IsAndroidAOT))]
```

Comment:
Same issue as the wasm one? Or wrong issue linked? If the failure is the exact same and for possibly the same underlying reason, can the issue be updated to include that it is problematic on Android as well? Or maybe its better to create a separate issue if its not clear they're failing for the same reasons.

Reply:
It's the same, I left a comment in the active issue